foundations/html_css/css-foundations/intro-to-css.md

```diff
@@ -405,10 +399,10 @@ This section contains questions for you to check your understanding of this less
 
 - <a class="knowledge-check-link" href="#class-selectors">What is the syntax for class and ID selectors?</a>
 - <a class="knowledge-check-link" href="#the-grouping-selector">How would you apply a single rule to two different selectors?</a>
-- <a class="knowledge-check-link" href="#chaining-selectors">Given an element that has an id of `title` and a class of `primary`, how would you use both attributes for a single rule?</a>
+- <a class="knowledge-check-link" href="#chaining-selectors"> Given an element that has an id of `title` and a class of `primary`, how would you use both attributes for a single rule?</a>
```

Comment:
I'm not entirely sure why there are two edits in this section for two knowledge checks, but the same edits were not made for the other checks. I think this is unrelated to the PR/issue so could safely be reverted back unless I'm missing something, please do tell me if so.

Reply:
Hey, @ManonLef I made the reverts. The comment is unrelated to this PR. I was making a mere observation on formatting. Should I raise a related issue about the formatting? I know TOP has a standard but it can be very confusing for contributors when markdown: lint points them out as errors.

Reply:
@CBID2 I think you can open an issue so the maintainers more versed in the style guide can chime in. However, I think it's safest to not let any linter or formatter edit files for projects that have a style guide.